contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -110,10 +104,10 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     /**
      * @dev See {ITeleporterMessenger-retrySendCrossChainMessage}
      *
-     * Emits {SendCrossChainMessage} event.
+     * Emits a {SendCrossChainMessage} event.
      * Requirements:
      *
-     * - `message` must have been previously sent to the given destination chain ID.
+     * - `message` must have been previously sent to the given `destinationChainID`.
```

Comment:
I think we should rename this to `destinationBlockchainID`, as per our other discussions of confusing this with the ethereum `chainID`.

Reply:
will echo this as DevRel brought it up too. They also suggsted possible `avalancheChainID`

Reply:
I definitely agree that we should rename these variables to `destinationBlockchainID`. Slight preference for that over `destinationAvalancheChainID` for verbosity.

Reply:
However, I think this is best suited for a standalone set of PRs, since it will require coordination on the awm-relayer side. Ticket: https://github.com/ava-labs/teleporter/issues/95

Reply:
sgtm thanks for making the issues